README.md

```diff
@@ -56,10 +56,11 @@ community](https://github.com/thoughtbot/factory_girl/graphs/contributors).
 Need Help?
 ----------
 
-We offer 1-on-1 coaching. We can help you set up factory_girl, clean up your
-`factories.rb`, or speed up slow tests. [Get in touch].
+We offer 1-on-1 [coaching]. We can help you set up factory_girl, clean up your
+`factories.rb`, or speed up slow tests. Or, [hire us] to build your project!
 
-[Get in touch]: http://coaching.thoughtbot.com/rails/?utm_source=github
+[coaching]: https://robots.thoughtbot.com/rails-coaching?utm_source=github
```

Comment:
The issue may simply be DNS (mis-)configuration; since the link to the blog post is also broken.  If so feel free to discard.